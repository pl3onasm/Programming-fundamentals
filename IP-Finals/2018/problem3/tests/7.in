173
535 525 341 355 793 580 740 484 249 101 96 93 28 438 799 20 182 503 684 551 374 182 705 527 108 756 655 943 186 684 218 339 979 291 440 618 272 341 271 574 934 25 130 533 669 953 298 340 41 954 836 714 164 444 294 89 954 30 109 705 611 218 25 793 711 430 149 535 954 363 580 286 853 844 303 595 611 542 652 331 294 700 794 68 218 291 538 322 402 746 738 132 3 954 919 340 618 941 948 102 173 226 98 622 43 377 68 46 424 820 589 289 351 123 594 186 201 395 374 759 468 734 908 109 903 738 158 763 873 622 378 55 290 219 930 395 614 746 104 495 186 123 249 635 917 331 331 954 438 784 319 495 506 588 351 363 485 589 594 99 830 378 819 355 675 291 925 589 303 14 430 469 278 364 173 499 387 830 83 339 675 172 821 119 873 355 530 750 269 98 158 182 717 821 838 46 485 740 122 372 47 535 290 977 821 738 756 397 842 538 953 488 519 68 954 367 402 261 728 788 676 551 150 867 269 28 411 830 543 530 121 372 619 46 839 928 4 613 311 19 588 700 289 836 676 291 927 775 213 676 254 862 387 913 90 411 754 351 535 943 953 386 687 316 36 55 884 160 108 101 174 530 174 444 150 65 367 311 189 309 938 788 530 977 828 714 506 355 341 458 956 186 658 499 158 839 635 740 611 55 618 937 808 611 172 821 423 577 356 158 356 90 675 411 788 30 574 149 83 796 462 699 797 367 194 355 610 104 844 96 30 79 962 194 269 395 36 756 712 458 742 150 269 930 79 278 55 372 402 266 930 213 483 218 493 806 914 613 820 917 977 927 368 854 739 662 881 669 38 367 121 714 904 954 999 583 19 687 469 611 455 438 262 186 1 339 853 977 119 1 669 212 985 363 269 56 687 372 568 378 907 808 566 438 928 402 19 322 341 186 321 378 240 652 402 249 389 96 296 652 553 356 218 611 760 676 491 119 271 440 687 538 714 740 756 708 249 505 661 291 832 189 332 387 262 452 742 687 356 319 792 127 213 919 590 411 53 108 682 8 252 395 821 585 462 558 4 903 483 595 423 746 102 555 462 321 86 979 141 201 830 742 212 331 648 150 763 903 194 917 568 976 588 1 174 298 792 675 286 838 182 738 295 655 109 260 355 808 312 685 927 523 588 93 577 832 536 930 340 90 734 928 249 99 174 594 213 136 353 979 876 954 158 669 597 695 461 238 941 619 290 655 675 794 28 701 673 499 25 363 455 705 865 295 941 524 830 79 56 530 619 504 89 522 976 301 90 182 273 16 189 542 142 794 201 262 614 123 493 363 377 739 549 597 275 833 188 877 773 19 600 493 120 773 734 862 717 309 508 675 269 295 96 687 245 577 752 248 928 99 716 941 218 415 797 598 635 472 580 342 491 775 469 266 808 816 873 537 444 290 881 958 573 716 537 635 205 99 588 956 720 275 295 549 846 803 261 364 652 368 16 440 488 716 568 294 739 756 455 648 319 701 811 734 76 278 402 618 293 377 251 86 240 662 98 421 4 676 740 275 469 189 127 319 425 41 219 249 127 36 20 4 336 251 341 127 600 589 605 740 938 295 867 907 342 551 582 440 508 817 127 121 913 720 261 842 598 289 377 555 820 588 424 803 336 20 286 316 528 981 655 162 199 122 378 618 205 506 119 819 342 349 538 742 99 808 411 977 275 230 98 542 8 985 440 573 341 846 699 740 799 186 395 386 319 825 853 935 948 102 186 687 873 750 218 573 594 830 108 590 483 999 935 290 281 839 938 298 372 79 811 979 684 830 55 966 676 523 142 389 956 397 684 540 83 368 839 495 699 332 55 655 793 919 685 319 740 205 839 913 281 76 86 199 705 339 370 708 41 962 919 127 174 669 351 127 341 298 150 123 530 598 62 954 295 295 682 331 488 14 927 817 455 320 28 271 828 595 917 252 305 269 530 411 372 779 806 833 585 699 76 793 252 305 928 821 326 772 341 779 716 367 272 999 705 424 716 865 530 519 899 872 41 148 661 16 213 462 505 729 682 20 917 530 322 526 402 981 262 55 943 109 605 149 537 917 594 594 485 167 760 934 658 884 598 55 149 514 734 285 503 493 904 675 164 605 884 36 186 102 798 355 738 353 296 252 535 551 854 430 750 598 801 540 356 293 286 553 183 937 127 467 372 148 589 305 119 838 47 262 927 669 368 218 266 938 463 816 711 342 342 759 573 917 173 853 240 899 275 417 577 873 254 792 127 295 739 519 715 979 0
