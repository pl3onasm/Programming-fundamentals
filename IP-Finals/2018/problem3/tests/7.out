1+172
25+148
41+132
43+130
46+127
53+120
65+108
83+90
